app/main.py

```diff
@@ -1,8 +1,50 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int | float,
+                 clean_mark: int | float,
+                 brand: int | float) -> None:
```

Comment:
The type annotations `int | float` are incorrect. In Python, you should use `Union[int, float]` from the `typing` module for type annotations that allow multiple types.